scss/_modal.scss

```diff
@@ -7,11 +7,6 @@
 .modal-open {
   // Kill the scroll on the body
   overflow: hidden;
-
-  .modal {
-    overflow-x: hidden;
-    overflow-y: auto;
-  }
```

Comment:
Dont remove these, because cause scrolling issue, see https://deploy-preview-30003--twbs-bootstrap.netlify.com/docs/4.3/components/modal/#scrolling-long-content and click the firse "Lanch demo modal".

Reply:
We can fix this by setting the code below on the "modal-body"
`.modal-body {
    overflow-y: auto;
    max-height: 65vh;
}`

Reply:
Or we can extend ".modal-dialog-scrollable" placeholder inside our modal-dialog by default. It won't affect anything.

Reply:
Modal scrolls entirely by default. There are no plans to change this behavior.

Reply:
But this behavior causes the issue in the video. and what about the vertical centering and IE?

Reply:
Removing L11-14 has nothing to do with the problem in your video. 

> 2- Remove ".modal {overflow-x: hidden;overflow-y: auto;}" it makes the modal content scrollable in mobile browsers if you tried to drag it.

It is a browser bug, see https://deploy-preview-30003--twbs-bootstrap.netlify.com/docs/4.3/getting-started/browsers-devices/#overflow-and-scrolling.